### PR TITLE
Feature Complete BookStore design

### PR DESCRIPTION
**Pull request by jubaer919**
# Highlight 🔍 

- Add any missing JSX elements to my project (without adding extra functionality).
- My final design is match Zeplin's design (this includes all elements + the same styling).
- I use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/components/books/Book.js [hunk 1]
```diff
@@ -1,31 +1,65 @@
...
+        </div>
+        <div className={classes.chapter}>
+          <p className={classes.current}>CURRENT CHAPTER</p>
+          <h3 className={classes['chapter-name']}>{chapterProgress}</h3>
+          <button type="button" className={classes['btn-update']}>UPDATE PROGRESS</button>
+        </div>
+      </div>
+    </div>
   );
-}
```
Comment (roshan-bajgain) [inline src/components/books/Book.js]: - [ ] As you should stick to the design as much as possible like color, same font, spacing. Please make sure that your design stick to the design as much as possible. Please see the design below for reference.
### Your design
his


### Original design
original

File: src/components/books/AddBook.js [hunk 2]
```diff
@@ -1,73 +1,66 @@
...
-          />
-          <input
-            type="text"
-            placeholder="Enter Author Name"
-            onChange={authorChangeHandler}
-            value={author}
+            placeholder="Enter Category"
+            onChange={categoryChangeHandler}
+            value={category}
+            className={classes['input-2']}
```
Comment (roshan-bajgain) [inline src/components/books/AddBook.js]: - [ ] You should stick to the design as much as possible like color, font and spacing. please compare your design with original and make the necessary changes. 
- [ ] Also make sure to add the same background color that is `#fafafa`
### Your design
his

### Original design
original

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @jubaer919, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- Gitflow is followed. ✔️
- Almost done ✔️ 

## Required Changes ♻️
- [ ] I cannot leave comment there so I am leaving it here. I would suggest you please make sure that you navbar matches the design like font, color, and spacing. see the image below for reference.
- [ ] Please `login to Zeplin` and use the credentials that is username and password and then you can style you app by using same color, font and spacing then you are good to go.
### Your design
hissssss

### Original design
his

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/books/Book.js (hunk 1 above)
Comment (jubaer919) [reply src/components/books/Book.js]: I don't have exact color they use css-variable

File: src/components/books/IndividualBook.module.css [hunk 3]
```diff
@@ -20,8 +22,103 @@
...
+
+.design > *:not(:last-child) {
+  border-right: 1px solid #eee;
+}
+
+.current {
+  font-size: 14px;
+  color: rgb(156, 152, 152);
+  margin-bottom: 2px;
+}
```
Comment (shella12) [inline src/components/books/IndividualBook.module.css]: - [ ] Great job on changing the color of your elements to match that of the provided design however you still need to fix the typography i.e. font-family, size, weight etc to match the provided design. Please login [login to Zeplin](https://app.zeplin.io/login) using this credentials
username: student@microverse.org
password: jb077mL2dL0DfxJ 
and checkout the typography to match the design.

File: src/components/books/AddBook.js [hunk 4]
```diff
@@ -1,73 +1,66 @@
...
           />
```
Comment (shella12) [inline src/components/books/AddBook.js:64]: - [ ] Please change the color of input borders to this ` #e8e8e8` to match the provided design as well.

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @jubaer919, 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. You are almost there...



## Highlights

- Added required color ✅ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/books/Book.js (hunk 1 above)
Comment (roshan-bajgain) [reply src/components/books/Book.js]: If you see at top you will get it. please login and see at top.

File: src/components/books/IndividualBook.module.css (hunk 3 above)
Comment (jubaer919) [reply src/components/books/IndividualBook.module.css]: i Can't find any font name robotoSlab

**Review by ademibowale — APPROVED** [review]
Hi @jubaer919 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️
- Styled the Book store applicatio  ✔️
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**